specification/v4.0/OSDM-online-api-v4.0.0-draft.yml

```diff
@@ -12996,15 +12996,15 @@ components:
       type: object
       additionalProperties: false        
       description: |
-              product selection by product code or product tag for an offer request. The number of items indicates the quantity to be required. Passengers must be referenced in case the product is intended for passengers.
-      properties:
-          oneOf:
-            - productCode:
-                 type: string
-                 examples:
-                   - PT00AD                    
-            - productTag:
-                 type: string
+              product selection by product code or product tag for an offer request. The number of items indicates the quantity to be required. 
+              Passengers must be referenced in case the product is intended for passengers. Eigther a prottag must be provided. 
```

Comment:
The comment is incomplete probably. You can use on from 3.7.0. I fixed this on Friday there.